R/websocket.R

```diff
@@ -6,6 +6,56 @@ NULL
 # Used to "null out" handler functions after a websocket client is closed
 null_func <- function(...) { }
 
+options <- list(max_message_size=32*1024*1024)
+#' Manage WebSocket options
+#'
+#' Lists the default options or assists in creating a custom configuration to
+#' use when creating a new WebSocket.
+#'
+#' @param ... Named options to be overridden.
+#' @details
+#' The following options are available:
+#' \describe{
+#'   \item{\code{max_message_size}}{The maximum size of a message in bytes. If a message
+#'   larger than this is sent, the connection will fail with the \code{message_too_big}
+#'   protocol error.}
+#' }
```

Comment:
Should mention somewhere what the default value is (and maybe why it's set to that?)